src/ngraph_encapsulate_op.cc

```diff
@@ -309,6 +309,60 @@ class NGraphEncapsulateOp : public OpKernel {
 
       auto function_size = ng_function->get_graph_size() / 1024;  // kb unit
 
+      // Get the serialized ops and stored the allreduce ops to a vector and
+      ng::NodeVector allreduce_op_list;
+      for (const shared_ptr<ng::Node>& node : ng_function->get_ordered_ops()) {
+        if (node->get_name().compare(0, 9, "AllReduce") == 0) {
```

Comment:
Can we use node->description() to get the type of op?
https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/node.hpp#L115